Lib/test/support/_hypothesis_stubs/__init__.py

```diff
@@ -0,0 +1,71 @@
+import functools
+import unittest
+
+__all__ = [
+    "given",
+    "example",
+    "assume",
+    "reject",
+    "register_random",
+    "strategies",
+    "HealthCheck",
+    "settings",
+    "Verbosity",
+]
+
+from . import strategies
+from ._settings import HealthCheck, Verbosity, settings
+from .control import assume, reject
```

Comment:
For stubs, I'd just define all of these inline in the `__init__` file - they should only ever be accessed via `hypothesis.foo` (and we should get on with https://github.com/HypothesisWorks/hypothesis/issues/2175) - but this certainly works.

Reply:
Sounds good.